docs/appendices/release-notes/5.0.0.rst

```diff
@@ -0,0 +1,205 @@
+.. _version_5.0.0:
+
+=============
+Version 5.0.0
+=============
+
+Released on 2022-07-11.
+
+.. NOTE::
+
+    If you are upgrading a cluster, you must be running CrateDB 4.0.2 or higher
+    before you upgrade to 5.0.0.
+
+    We recommend that you upgrade to the latest 4.8 release before moving to
+    5.0.0.
+
+    A rolling upgrade from 4.8.x to 5.0.0 is supported.
+
+    Before upgrading, you should `back up your data`_.
+
+.. _back up your data: https://crate.io/docs/crate/reference/en/latest/admin/snapshots.html
+
+
+
+.. rubric:: Table of Contents
+
+.. contents::
+   :local:
+
+Breaking Changes
+================
+
+- Changed the type of :ref:`TIMESTAMP <type-timestamp>` from :ref:`TIMESTAMP WITH
+  TIME ZONE <type-timestamp-with-tz>` to :ref:`TIMESTAMP WITHOUT TIME ZONE
```

Comment:
Almost forgot about this but I think we should add an additional entry that  mentions the Lucene update and that tables created before CrateDB 4.0 are no longer supported. Thats the main reason for the version bump to 5 after all.

Reply:
done in https://github.com/crate/crate/pull/12737/commits/12b1757d800e80391a551712316a24df681881e0